src/app/frontend/replicasetdetail/replicasetdetail_controller.js

```diff
@@ -156,6 +165,19 @@ export default class ReplicaSetDetailController {
    */
 
   /**
+   * Updates replicas count in replica set
+   * @param {number} replicasCount
+   * @param {function(!backendApi.ReplicaSetDetail)=} opt_callback
+   * @param {function(!angular.$http.Response)=} opt_errback
+   * @private
+   */
+  updateReplicas_(replicasCount, opt_callback, opt_errback) {
+    this.replicaSetDetail.podsDesired = replicasCount;
+    this.replicaSetDetailResource_.save(this.replicaSetDetail, opt_callback, opt_errback);
+    // TODO(floreks): Think about refreshing data on this page after update.
```

Comment:
We potentially could, but dont do this in this PR. Remember, the smaller PRs the better :) Open an issue about how to synchronize changes on modification.

For a nice start we can do resource refresh after a few seconds. This should work in most cases.